docs/contributing/blog-and-website-contributions.md

```diff
@@ -1,71 +1,71 @@
 ---
-title: Blog & Website Contributions
+title: Contribuições ao Blog & Website
 ---
 
-We wholeheartedly welcome contributions to the Gatsby blog and website!
+Nós sinceramente agradecemos as contribuições para o blog e o site do Gatsby! 
 
-Here are some things to keep in mind when deciding where to contribute to Gatsby:
+Aqui estão algumas coisas a ter em mente ao decidir para onde contribuir com o Gatsby:
 
-- [Blog posts](#contributing-to-the-blog) work best for case studies and time-sensitive storytelling (see the [blog post format](#blog-post-format)).
-- [Docs](/contributing/docs-contributions/) are continually relevant and discoverable learning materials that go beyond any one case study or situation.
-- [Website changes](#making-changes-to-the-website) that improve either of these are always welcome!
+- [Postagens de Blog](#contribuindo-para-o-blog): funcionam melhor para estudos de caso e narrações sensíveis ao tempo (veja o [formato de postagem do blog](#formato-de-postagem-do-blog)).
+- [Docs](/contributing/docs-contributions/): são materiais de aprendizagem continuamente relevantes e notáveis, que vão além de qualquer estudo de caso ou outra situação.
+- [Modificações do Website](#fazendo-modificações-para-o-website): melhorias aqui são sempre bem-vindas!
```

Comment:
A tradução de "making-changes-to-the-website" não seria melhor "fazendo modficações no site"?